docs/proposals/kubelet-qos.md

```diff
@@ -0,0 +1,142 @@
+# Quality of Service in Kubernetes
+
+**Author**: Ananya Kumar (@AnanyaKumar)
+
+**Status**: Draft proposal; prototype in progress.
+
+*This document presents the design of quality of service tiers for pods in Kubernetes, describes use cases, and lays out milestones for its development.*

```

Comment:
Or Resource QoS classes, as discussed in #11967.

The main problems with "tier" are:
1. To people outside Google it has a totally different meaning: https://en.wikipedia.org/wiki/Multitier_architecture
2. To Googlers, it brings a lot of baggage, including preemption priority.